apps/src/studio/levels.js

```diff
@@ -244,13 +244,13 @@ levels.playlab_1 = utils.extend(levels.dog_hello, {
   ]
 });
 
-levels.iceage_1 = utils.extend(levels.playlab_1, {
+levels.iceage_hello1 = utils.extend(levels.playlab_1, {
   background: 'iceberg',
-  firstSpriteIndex: 0, // manny
+  firstSpriteIndex: 1,
```

Comment:
I noticed that most of them were wrong, and I think it would be better to go back and make these enums or use different syntax that makes it more clear